maxdiff/print_unicode.py

```diff
@@ -0,0 +1,14 @@
+"""
+Module that shares a function to gracefully handle printing unicode characters 
+if the user has the PYTHONIOENCODING environment vairable set to 'ascii'. 
+In that case, replace any characters that can't be encoded with ?.
+"""
+
+
+def print_unicode_string(s):
+    try:
+        encoded_string = s.encode("utf-8", "replace")
+        print(s)
```

Comment:
Don't we want to return the now encoded string? Not the original?

Reply:
Ah you are absolutely right, the encode statement isn't needed there. The try already fails if we try to print s when it contains non-ascii characters and PYTHONIOENCODING is set to ascii. I'll fix this.